e2e/vm/lifecycle_test.go

```diff
@@ -50,6 +57,12 @@ var testVMLifecycle = func(o *option.Option) {
 				command.New(o, virtualMachineRootCmd, "remove").WithTimeoutInSeconds(60).Run()
 				command.RunWithoutSuccessfulExit(o, "images")
 			})
+			ginkgo.It("should be able to force remove the virtual machine", func() {
+				command.New(o, virtualMachineRootCmd, "init").WithTimeoutInSeconds(600).Run()
+				command.New(o, virtualMachineRootCmd, "stop").WithTimeoutInSeconds(60).Run()
+				command.New(o, virtualMachineRootCmd, "remove", "--force").WithTimeoutInSeconds(60).Run()
+				command.RunWithoutSuccessfulExit(o, "images")
+			})
```

Comment:
Why do we need two `should be able to force remove the virtual machine`?

Reply:
force remove should work in any state (apart from nonexistent) so I wanted to cover both "Running" and "Stopped". perhaps it makes more sense for this test to just be force stop, though

Reply:
Yeah, I feel we can just make it `stop --force` instead and move it to the "the virtual machine is in running status" `When` block.

> force remove should work in any state (apart from nonexistent) so I wanted to cover both "Running" and "Stopped".

I feel testing `remove --force` for `Stopped` may not be necessary as a stopped VM should be able to be removed with `--force` in the first place, but I'm also fine with having it.